src/main/java/com/answerdigital/answerking/controller/TagController.java

```diff
@@ -89,11 +91,32 @@ public ResponseEntity<TagResponse> addTag(@Valid @RequestBody final TagRequest t
                 content = {
                     @Content(mediaType = "application/problem+json", schema = @Schema(implementation = ErrorResponse.class))})
     })
-    @PostMapping(path = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @PutMapping(path = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<TagResponse> updateTag(
             @Valid @PathVariable final Long id,
             @Valid @RequestBody final TagRequest tagRequest
     ) {
         return new ResponseEntity<>(tagService.updateTag(id, tagRequest), HttpStatus.OK);
     }
+
+    @Operation(summary = "Delete an existing tag.")
```

Comment:
Can we please change this to 'Retire an existing tag.'? It hasn't been defined in the spec like this, but the spec will need updating.